eng/scripts/Get-BinarySizes.ps1

```diff
@@ -41,13 +41,16 @@ function setEnvVar($key, $value) {
 }
 
 function getTargetOs {
-    if ($OsVMImage.StartsWith('macOS')) {
+    if ($OsVMImage.StartsWith('macOS', $true, (Get-Culture).InvariantCulture)) {
```

Comment:
This is fine but FYI here is another powershell mechanism I find to be cleaner:

```
if ($OsVMImage -like "macos*") {
```

Reply:
Oooh, that's way better.

Reply:
I'll update the script in a future pull request.